### PR TITLE
[ETFE-2174] guarantor required page

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: app/pages/sections/guarantor/GuarantorRequiredPage.scala [hunk 1]
```diff
@@ -14,19 +14,14 @@
...
 
-import pages.behaviours.PageBehaviours
+import pages.QuestionPage
+import play.api.libs.json.JsPath
 
-class ConsigneeExcisePageSpec extends PageBehaviours {
+case object GuarantorRequiredPage extends QuestionPage[Boolean] {
 
-  "ConsigneeExcisePage" - {
+  override def path: JsPath = JsPath \ toString
```
Comment (grahampaulcook) [inline app/pages/sections/guarantor/GuarantorRequiredPage.scala:24]: `override def path: JsPath = JsPath \ "guarantor" \ toString`

File: app/navigation/GuarantorNavigator.scala
Comment (grahampaulcook) [inline app/navigation/GuarantorNavigator.scala:1]: Missing a `GuarantorNavigatorSpec` for this.

File: conf/guarantor.routes
Comment (grahampaulcook) [inline conf/guarantor.routes:1]: To be consistent with the other section routes this should be changed to `/guarantor/required`

File: app/views/sections/guarantor/GuarantorRequiredView.scala.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+        )
+
+        @govukButton(
+            ButtonViewModel(messages("site.continue"))
+        )
+    }
+}
+
+@heading = {
+    @h1("guarantorRequired.heading", Some(messages("guarantor.subHeading", request.lrn)), "govuk-fieldset__heading")
```
Comment (grahampaulcook) [inline app/views/sections/guarantor/GuarantorRequiredView.scala.html:52]: I believe we need to take the h2 out of legend due to the WCAG 2.1 violation ( see https://jira.tools.tax.service.gov.uk/browse/ETFE-2004 )

File: app/pages/sections/guarantor/GuarantorRequiredPage.scala (hunk 1 above)
Comment (james-work-account) [reply app/pages/sections/guarantor/GuarantorRequiredPage.scala:24]: Good spot. Probably not for this PR, but should this change be added to the `move2section.sh` script?

Comment (grahampaulcook) [reply app/pages/sections/guarantor/GuarantorRequiredPage.scala:24]: We could do, although that `move2section.sh` is quite basic at the moment of moving stuff to packages and not doing and processing of the files.

**Review by grahampaulcook — APPROVED** [review]

**Pull request merged**